TestResults/results.csv

```diff
@@ -0,0 +1,36 @@
+Function_Name,Total time taken in ms,Total time taken in ns,No. of Iterations,Average Latency in ns,TPS 
```

Comment:
Remove "number of iterations". Since this the number of iterations is fixed for all functions we do not need this as a column. We can include the number of iterations in the readme for the CSV we upload. 

Change head name Total time taken in ms to "Total Time (ms)"
The time recorded should appear as 5465.34 currently the decimal points are missing

Remove "Time Taken in ns" col. This is not required.

Change col name "Average Latency in ns" to Average Latency (ns). Only use two decimal points when you report (need to truncate or round)

Change col name TPS to Throughput (operations/second) (Note the unit)

Reply:
Remove "Time Taken in ns" col. This is not required.

we need to calculate the total time in millisecond or nano second?

I was planning to do in nanosecond with nanoTime() function. just added the millisecond for a comparison between both.